src/services/validatorSchema.ts

```diff
@@ -0,0 +1,16 @@
+import * as yup from 'yup';
+
+export const inventoryItemSchema = yup.object({
+    inventory_address: yup.string().typeError("inventory_address must be a string").defined("The inventory address must be provided, don't forget it ! "),
```

Comment:
same comments here as in the `item-store` PR:
- Let's make the error message even more explicit and include the expected type for the field.
- let's make the type stricts, for instance if a number is provided let's still have the validation fail even if it could be converted to a string, this would prevent issues later on when we tokenize items